dev/com.ibm.ws.config/resources/com/ibm/ws/config/internal/resources/ConfigMessages.nlsprops

```diff
@@ -476,5 +476,10 @@ error.root.must.be.server=CWWKG0105E: The {0} configuration document must begin
 error.root.must.be.server.explanation=The configuration must be contained in either a server element or a client element. 
 error.root.must.be.server.useraction=Add a server or client element as the root of the configuration file. 
 
+error.bad.variable.file=CWWKG0106E: The file {0} can not be read. 
+error.bad.variable.file.explanation=The file can not be loaded as a variable because it can not be read. 
```

Comment:
is it possible to say why the file can't be read or what cannot read it?

Reply:
Not really.. We just know that something went wrong. The most likely causes are in the user action.